Python/import.c

```diff
@@ -387,11 +387,32 @@ import_get_module(PyThreadState *tstate, PyObject *name)
 }
 
 
-PyObject *
-PyImport_GetModule(PyObject *name)
+static int
+import_ensure_initialized(PyThreadState *tstate, PyObject *mod, PyObject *name)
 {
-    PyThreadState *tstate = _PyThreadState_GET();
-    return import_get_module(tstate, name);
+    PyInterpreterState *interp = tstate->interp;
+    PyObject *spec;
+
+    _Py_IDENTIFIER(__spec__);
```

Comment:
Please move both these statics to the top of the file (next to `CACHEDIR`) as globals.

Reply:
I think it would be better to make that as a separate change. This style where they're block-local is the way the existing code works, which keeps the diff simple.

It's also the way that every use of `_Py_IDENTIFIER` in this file works. So I think the cleanest way to change the style would be to sweep through this whole file (at least) in one PR, separate from any substantive changes.

Reply:
I agree with @gnprice . To keep this diff simple, am opening a separate [bpo issue](https://bugs.python.org/issue37872) to track this refactor. I will work on it If this PR is merged.

Reply:
sounds good